pkg/util/informermanager/federatedinformermanager.go

```diff
@@ -276,6 +282,17 @@ func (m *federatedInformerManager) processCluster(
 		}
 	}
 
+	registrations := m.podEventRegistrations[clusterName]
+	factory := m.informerFactories[clusterName]
+	for _, handler := range m.podEventHandlers {
+		if registrations[handler] == nil {
+			copied := handler.copyWithClusterName(clusterName)
```

Comment:
Why is this copy needed?

Reply:
need a clusterName within for every pod eventHandler to each cluster